server.js

```diff
@@ -247,7 +252,7 @@ app.use((err, req, res, next) => {
   res.status(500).send('Something broke!');
 });
 
-app.listen(6500 || process.env.PORT || conf.PORT);
+app.listen(8000 || process.env.PORT || conf.PORT);
```

Comment:
```suggestion
app.listen(conf.PORT);
```
I don't think we'll be working at the same time anymore. Also, is there a need for `process.env.PORT`?

Reply:
```suggestion
app.listen(conf.PORT || 8000);
```

Reply:
I think this is a part from a legacy code. Let me knwo if this change makes sense to you